src/net/WebRTC/WebRTCWebSocketPeer.cs

```diff
@@ -138,8 +138,15 @@ protected override async void OnOpen()
                 // Don't log SDP can contain sensitive info, albeit very short lived.
                 //logger.LogDebug(offerSdp.sdp);
 
-                Context.WebSocket.Send(offerSdp.toJSON());
+                try
+                {
+                    Context.WebSocket.Send(offerSdp.toJSON());
+                }
+                catch (Exception ex)
+                {
+                    logger.LogError($"An error has occurred during the OnOpen event.\n{ex.ToString()}.");

```

Comment:
Swallowing the exception here could be problematic for existing implementations. Is it not possilbe to catch the exception in your application code?

Reply:
The issue was explained in https://github.com/sipsorcery-org/sipsorcery/issues/1120#issuecomment-2111552802. The exception thrown by OnOpen() become unhandled exception due to it being an async void function. Due to this it crashed the application.

Any advice on resolving this on the application code?